.github/workflows/ci-pytorch-dockers.yml

```diff
@@ -8,8 +8,9 @@ on:
     paths:
       - "dockers/**"
       - "!dockers/README.md"
-      - "requirements/**"
       - "requirements.txt"
+      - "requirements/*.txt"
```

Comment:
Tangentially related to this PR, I think the paths [here](https://github.com/Lightning-AI/lightning/blob/acd4805f1a284e513272d150de6f98f27a0489b3/.github/checkgroup.yml#L75) should include the requirements